src/assets/css/nav.css

```diff
@@ -1,5 +1,33 @@
 /* NAV */
 /* overide bootstrap mobile nav styles */
+
+/* skip to main link - Accessibility */
+
+#skip-to-main-link {
+  font-size: 1.3rem;
+  left: 10px;
+  background-color: var(--primary-1);
+  border: 1px solid var(--element-3);
+  border-radius: 0 0 4px 4px;
+  padding: 12px 24px;
+  color: var(--element-4);
+  font-weight: 700;
+  text-transform: capitalize;
+  text-decoration: none;
+  position: absolute;
+  transform: translateY(-120%);
+  transition: transform 325ms ease-in;
+  z-index: 99;
+}
+
+#skip-to-main-link:hover {
```

Comment:
I think we have styles that define an u decline already in the project. Could we reuse those instead of creating a new rule for the same effect?